config/services.go

```diff
@@ -0,0 +1,19 @@
+package config
+
+// Default endpoints to access services. These endpoints are overritten by the build
+// Use the following format for the variable name: "[Service]URL" (where Service is the name of the service capitalized)
+// The service name should be the name of the directory inside `systemservices` but capitalized
+// example:
+// 		var (
+// 			FooURL string
+// 			BarURL string
+// 		)
+var ()
+
+func (c *Config) initServices() {
```

Comment:
What is the purpose of initServices?

Reply:
It's called from the config, it's to have all the logic about the services in the same folder instead of having to initialize it on the config.go but declare it here